recipe/meta.yaml

```
@@ -103,6 +103,9 @@ test:
     # Check for bdist_conda
     - python test_bdist_conda_setup.py bdist_conda --help
 
+  downstreams:
+    - conda-smithy
```

```suggestion
    - conda-smithy  # [py >=3]
```

So we don't run tests on old python